src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -790,9 +805,7 @@ long intensityBasedDelayMillis(double intensity) {
       long repairDuration = Math.max(1, repairEnd - repairStart);
       long delay = (long) (repairDuration / intensity - repairDuration);
       LOG.debug("Scheduling next runner run() with delay {} ms", delay);
-      int nbRunningReapers = countRunningReapers();
-      LOG.debug("Concurrent reaper instances : {}", nbRunningReapers);
-      return delay * nbRunningReapers;
+      return delay;
```

Comment:
The number of concurrent reaper instances isn't relevant anymore (if it ever was). We're now keeping the lock on cluster nodes which gives us the expected behavior.